Doc/library/importlib.rst

```diff
@@ -1285,65 +1289,70 @@ find and load modules.
    typically exposed as the module's ``__spec__`` attribute.  In the
    descriptions below, the names in parentheses give the corresponding
    attribute available directly on the module object.
-   E.g. ``module.__spec__.origin == module.__file__``.  Note however that
-   while the *values* are usually equivalent, they can differ since there is
-   no synchronization between the two objects.  Thus it is possible to update
-   the module's ``__path__`` at runtime, and this will not be automatically
-   reflected in ``__spec__.submodule_search_locations``.
+   E.g., ``module.__spec__.name == module.__name__``.  Note however that
```

Comment:
```suggestion
   E.g., ``module.__spec__.origin == module.__path__``.  Note however that
```

Reply:
@brettcannon Thank you for the review. I accepted all your suggestions, except this one: ``module.__spec__.origin == module.__path__``, because:

1. This expression is false but I think you made a typo and you meant ``module.__spec__.submodule_search_locations == module.__path__``.
2. This example illustrates the _previous_ sentence that states the equivalence between spec attributes and module attributes in parentheses; it does not illustrates the _next_ sentence that states that two equivalent attributes are not synchronized, which is already illustrated by another example in the last sentence. So the pair of attributes in this example and in the next sentence don't need to be identical. The proof is that this example was (before this pull request) ``module.__spec__.origin == module.__file__`` and the last sentence was about a different pair of attributes: ``__path__`` and ``__spec__.submodule_search_locations``. However I agree that it might be more readable if the previous and next sentences are illustrated by the same example. So I committed your suggestion with the correction 1.

Reply:
You're correct I messed up my suggested edit.

I still think the examples should be in sync, though. Whether it's with `__path__`, `__file__`, or whatever isn't critical, but the sudden shift in example is a bit jarring and the wording doesn't prepare the reader for the shift.

Reply:
@brettcannon I agree. The examples are in sync now. Let me know if that is fine for you.